.github/workflows/node-ci-pr.yml

```diff
@@ -21,21 +21,10 @@ jobs:
         os: [ubuntu-latest]
         node-version: [16.x]
```

Comment:
This has the `node-version` so it would create a cache key starting with `Linux16.x-`